.github/workflows/build.yml

```diff
@@ -0,0 +1,118 @@
+# This file is autogenerated by maturin v1.7.4
+# To update, run
+#
+#    maturin generate-ci github --platform linux --platform macos
+#
+# manual edits:
+#    -  to limit the number of ubuntu variants
+#    -  do the rust build step
+#    -  to add the RUSTFLAGS environment variable
+#    -  to specify python version used by maturin
+
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+      - master
+    tags:
+      - '*'
+  pull_request:
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  linux:
+    runs-on: ${{ matrix.platform.runner }}
+    strategy:
+      matrix:
+        platform:
+          - runner: ubuntu-20.04
```

Comment:
let's just use ubuntu latest for this.  20.04 uses a pretty old GLIBC

Reply:
I tried, `ubuntu-latest` was the cause of the rust wrapper build break.
I suspect something in 22.04 or 24.04 is causing trouble.